docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -0,0 +1,197 @@
+            | Reject           | Reset traffic                   |
+            +------------------+---------------------------------+
+            | RedirectToUrl    | Redirect                        |
+            +------------------+---------------------------------+
+            | RedirectToPool   | Forward traffic to pool         |
+            +------------------+---------------------------------+
+
+Since the L7 Rules are expected to match on each HTTP request, if a listener has an L7Policy attached, the presence of a OneConnect profile on the corresponding TMOS virtual server must be assured.
+
+Initially, the support for Regex is not available.  Since there is a direct mapping of the other compare types in Neutron to BIG-IP LTM, and regular expressions can in some cases be expressed as the logical ‘OR’ of multiple string compares, we propose an initial implementation of L7 Policies and Rules without regular expressions with the understanding that it will be available as a feature in a future release.
+
+Prerequisites
+-------------
+
+- Licensed, operational BIG-IP :term:`device` or :term:`device cluster`.
+- Operational OpenStack cloud (|openstack| release).
+- Administrator access to both BIG-IP device(s) and OpenStack cloud.
+- F5 :ref:`agent <agent:home>` and :ref:`service provider driver <Install the F5 LBaaSv2 Driver>` installed on the Neutron controller and all other hosts from which you want to provision LBaaS services.
+- Knowledge of `OpenStack Networking <http://docs.openstack.org/mitaka/networking-guide/>`_ concepts.
+- Knowledge of BIG-IP `system configuration`_, `local traffic management`_, & `device service clustering`_.
+
+Caveats
+-------
+
+- As mentioned previously, the REGEX comparison type is not available in the current release.
+
+Configuration
+-------------
+
+#. No additional configuration needed in the :ref:`Agent Configuration File`.
+
+#. CLI Example (copied from the Neutron L7 feature page linked above):
+
+    .. code-block:: text
+        :emphasize-lines: 2,4,6,8,10
+
+        # Create a listener
+        neutron lbaas-create-listener listener1
+        # Create a pool
+        neutron lbaas-create-pool pool1
+        # Create a policy
+        neutron --policy policy1 lbaas-create-l7policy --name "policy1" --listener "listener1" --action redirect_to_pool --pool "pool1" --position 1
+        # Create a rule for this policy
+        # Once the below operation has completed, a new policy will exist on the device called 'wrapper_policy'.
+        # It will have a single rule called redirect_to_pool_1.
+        # A single condition and a single action will exist.
+        neutron lbaas-create-l7rule rule1 --rule-type path --compare-type starts_with --value "/api" --policy policy1
+        # Create a second rule for the above policy
+        neutron lbaas-create-l7rule rule2 --rule-type header --compare-type equal_to --key "mycookie" --value "myvalue" --invert --policy policy1
+
```

Comment:
Yeah, I think it'd be helpful to show exactly what these commands do on the BIG-IP, especially since the terminology is conflicting. The `tmos show` output for the object should do it, but check with @richbrowne on that -- he's the expert!